docs/source/releases/v1_12_2a0.rst

```diff
@@ -13,20 +13,84 @@
 Version 1.12.2a0 (2024-02-21)
 *****************************
 
+
 * Bug fixes
 
   * Ensure that log.interactive works from plugins when imported independently.
   * Make add_log_level a callable class to fix AttributeErrors.
+  * Fix product source_name duplicate entries bug
+
+* Testing Updates
+
+  * Added all_product_families.yaml to test all available product families
```

Comment:
Adding some documentation on the product families as well would be good - even if it is just pointing to the test scripts as examples.